requirements/Makefile

```diff
@@ -1,27 +1,6 @@
-.PHONY: all check clean help
+#
+# Targets to pip-compile requirements
+#
+include base.Makefile
```

Comment:
What is the purpose of splitting the content of this makefile in a different file if it is e.g. not reused?

Reply:
Sylvain set it up like this, good point. He probably had plans for it but never was able to tackle them? ^^